web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSecurityGroups.tsx

```diff
@@ -242,11 +239,16 @@ function getTips(
 }
 
 function allowsOutbound(sg: SecurityGroup): boolean {
-  return sg.outboundRules.some(rule => {
-    const havePorts = allowsOutboundToPorts(rule);
-    // this is a heuristic, because an exhaustive analysis is non-trivial.
-    return havePorts && rule.cidrs.some(cidr => cidr.cidr === '0.0.0.0/0');
-  });
+  return (
+    sg.outboundRules?.some(rule => {
+      if (!rule) {
+        return false;
+      }
+      const havePorts = allowsOutboundToPorts(rule);
+      // this is a heuristic, because an exhaustive analysis is non-trivial.
+      return havePorts && rule.cidrs?.some(cidr => cidr.cidr === '0.0.0.0/0');
```

Comment:
this too `cidrs?`

Reply:
removed - you're right this can't be undefined